webui/react/src/pages/ExperimentDetails/ExperimentInfoBox.tsx

```diff
@@ -101,7 +101,7 @@ const ExperimentInfoBox: React.FC<Props> = ({ experiment, onChange }: Props) =>
       label: 'Duration',
     },
     {
-      content: <Link isButton path={`/experiments/${experiment.id}/model_def`}>
+      content: <Link external isButton path={`/experiments/${experiment.id}/model_def`}>
```

Comment:
**blocker:** this resulted in a `{"message":"Unauthorized"}` response. Tried it after logging out and logging back in (in case it needed the cookie to be set.)

Reply:
FYI, on dev, the URL format looks good...
```
http://localhost:8080/experiments/62/model_def
```

Reply:
> blocker: this resulted in a {"message":"Unauthorized"} response. Tried it after logging out and logging back in (in case it needed the cookie to be set.)

I just checked release 0.13.6 and model_def download through localhost:3000 just redirects to root

Reply:
yep confirming 8080 works. Just putting a note here that this is another good reason to move into a pattern of using signed URL approach for downloading files, to avoid the domain-mismatch issue.